agot-bg-game-server/src/client/game-state-panel/utils/SupplyTrackComponent.tsx

```diff
@@ -27,7 +27,7 @@ export default class SupplyTrackComponent extends Component<SupplyTrackComponent
                             Each supply track column contains a list of numbers illustrating the
                             maximum size and count of each player&apos;s armies. (An army is made up
                             of two or more units within the same land or sea area.) The supply
-                            track is updated from game effects, such as the &quotSupply&quot Westeros
+                            track is updated from game effects, such as the "Supply" Westeros
```

Comment:
Please change it to `&quot;`. That should solve the issue and satisfy tslint…

Reply:
Done. Worked great!